CONTRIBUTING.md

```diff
@@ -62,7 +62,16 @@ included in the project:
    git checkout -b <topic-branch-name>
    ```
 
-4. Install the [Jekyll](https://github.com/jekyll/jekyll/) gem to preview locally.
+4. Install the [Jekyll](https://github.com/jekyll/jekyll/) gem and dependencies to preview locally:
+
+    ```bash
+    # Install the needed gems
+    gem install jekyll jekyll-sitemap pygments.rb
```

Comment:
With all due respect, this is not going to be an argument. I understand the pros and cons entirely and will continue to use the Gemfile.lock in this project.

If you’d like to update the CONTRIBUTING.md to reflect the bundler usage I’ll be happy to merge it. 

Thanks for your continued help! 